opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -192,6 +201,17 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
     }
 
     fn shutdown(&self) -> LogResult<()> {
+        let dropped_logs = self.dropped_logs_count.load(Ordering::Relaxed);
+        let max_queue_size = self.max_queue_size;
+        if dropped_logs > 0 {
+            otel_warn!(
+                name: "BatchLogProcessor.LogsDropped",
+                dropped_logs_count = dropped_logs,
+                max_queue_size = max_queue_size,
+                message = "Logs were dropped due to a full or closed queue. The count represents the total count of lost logs in the lifetime of the BatchLogProcessor. Consider increasing the queue size and/or decrease delay between intervals."
```

Comment:
```suggestion
                message = "Logs were dropped due to a queue being full or other error. The count represents the total count of lost dropped in the lifetime of this BatchLogProcessor. Consider increasing the queue size and/or decrease delay between intervals."
```

Given this is for end-users, I think we cannot use "closed queue", as that is strictly internal implementation that we use a channel which has the possibility of being closed. See if my above suggestion is good. Happy to get other feedback as well.
Also, we can improve the messaging in the future also based on feedbacks we get, so no need to block this PR.

Reply:
LGTM. I defer entirely to your experience on the way we speak to users!